src/modules/cpuUsage.c

```diff
@@ -1,21 +1,55 @@
 #include "fastfetch.h"
 #include "common/printing.h"
 #include "detection/cpuUsage/cpuUsage.h"
+#include "sys/time.h"
 
 #define FF_CPU_USAGE_MODULE_NAME "CPU Usage"
 #define FF_CPU_USAGE_NUM_FORMAT_ARGS 1
 
+time_t getTimeInMs()
+{
+    struct timeval timeNow;
+    gettimeofday(&timeNow, NULL);
+    return (timeNow.tv_sec * 1000) + (timeNow.tv_usec / 1000);
+}
+
+static long inUseAll1, totalAll1, startTime;
+
+void ffPrepareCPUUsage()
+{
+    startTime = getTimeInMs();
+    ffGetCpuUsageInfo(&inUseAll1, &totalAll1);
+}
```

Comment:
This doesn't seem to be time expensive on my system, maybe we can just always start it somewhere in ffStart